test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -411,6 +414,42 @@ def printSummary(self):
         self.logger.info('\n'.join(msg))
 
 
+class system_availability:
+    def __init__(self, hostname=None):
+        # getting RAM size
+        ssh = subprocess.Popen(['ssh', hostname, 'cat', '/proc/meminfo'],
+                               stdout=subprocess.PIPE)
+        meminfo = dict((i.split()[0].rstrip(':'), int(i.split()[1]))
+                       for i in ssh.stdout)
+        self.system_ram = meminfo['MemAvailable'] / (2**10)
+
+        # for home and exec size
+        home_disk = []
+        pbsconf_home = Server().pbs_conf['PBS_HOME']
+        pbsconf_exec = Server().pbs_conf['PBS_EXEC']
+        # getting mounte point for pbs home
+        home_info = subprocess.Popen(['ssh', hostname, 'df -k', pbsconf_home],
+                                     stdout=subprocess.PIPE)
+        home_disk.extend((i.split())
+                         for i in home_info.stdout)
+        home_size = home_disk[1][3]
+        home_mount = home_disk[1][5]
+        # getting mounte point for pbs exec
+        exec_disk = []
+        exec_info = subprocess.Popen(['ssh', hostname, 'df -k', pbsconf_exec],
```

Comment:
Why do we need to check PBS_EXEC size? Do we write anything in PBS_EXEC? I think checking PBS_HOME size will be sufficient?